### ISSUE TITLE
#region should not be displayed

### ISSUE THREAD

**Issue by RickAndMSFT** [dotnet/AspNetCore.Docs#22033]
First code example

```
// Unused usings removed.
using Microsoft.AspNetCore.Mvc.RazorPages;
using Microsoft.EntityFrameworkCore;
using RazorPagesMovie.Models;
using System.Collections.Generic;
using System.Threading.Tasks;

namespace RazorPagesMovie.Pages.Movies
{
    #region snippet1
    public class IndexModel : PageModel
    {
        private readonly RazorPagesMovie.Data.RazorPagesMovieContext _context;

        public IndexModel(RazorPagesMovie.Data.RazorPagesMovieContext context)
        {
            _context = context;
        }
        #endregion
        public IList<Movie> Movie { get;set; }

        public async Task OnGetAsync()
        {
            Movie = await _context.Movie.ToListAsync();
        }
    }
}
```

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 70f6104a-a223-4bb3-4101-f553002b9b26
* Version Independent ID: 15e79e80-8d46-7cd5-c883-6cc0f5940a1b
* Content: [Part 3, scaffolded Razor Pages](https://docs.microsoft.com/en-us/aspnet/core/tutorials/razor-pages/page?view=aspnetcore-5.0&tabs=visual-studio)
* Content Source: [aspnetcore/tutorials/razor-pages/page.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/tutorials/razor-pages/page.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-tutorials**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by fiyazbinhasan** [comment]
@RickAndMSFT I think updating the relative URL to `razor-pages-start/snapshot_sample5` will remove the region. What do you think?

**Comment by Rick-Anderson** [comment]
You fix it by adding an outer region.

```
#region snippetFull
// Unused usings removed.
using Microsoft.AspNetCore.Mvc.RazorPages;
using Microsoft.EntityFrameworkCore;
using RazorPagesMovie.Models;
using System.Collections.Generic;
using System.Threading.Tasks;

namespace RazorPagesMovie.Pages.Movies
{
    #region snippet1
    public class IndexModel : PageModel
    {
        private readonly RazorPagesMovie.Data.RazorPagesMovieContext _context;

        public IndexModel(RazorPagesMovie.Data.RazorPagesMovieContext context)
        {
            _context = context;
        }
        #endregion
        public IList<Movie> Movie { get;set; }

        public async Task OnGetAsync()
        {
            Movie = await _context.Movie.ToListAsync();
        }
    }
}
#endregion
```

Then change the snippet reference to `....?name=snippetFull`

**Comment by fiyazbinhasan** [comment]
maybe we should close this? @Rick-Anderson

**Comment by Rick-Anderson** [comment]
Fixed in #22073
